src/implementations/vanilla/components/link/_link-print.scss

```diff
@@ -14,13 +14,15 @@ $_negative-color: null !default;
   color: $_color;
   page-break-inside: avoid;
   text-decoration: none;
-}
 
-.ecl-link::after {
-  content: '(' attr(href) ')';
-  font-style: italic;
-  margin-inline-start: map.get(theme.$spacing-print, '2xs');
-  text-decoration: underline;
+  &::after {
+    color: map.get(theme.$color, 'black-100');
```

Comment:
we should set it to white for negative links
![image](https://github.com/ec-europa/europa-component-library/assets/25579221/f78e58d9-d384-46e8-af19-b403c68aac87)